.github/workflows/crux-mir-build.yml

```diff
@@ -243,6 +243,10 @@ jobs:
         with:
           submodules: true
 
+      - name: Generate crux.buildinfo.json
+        shell: bash
+        run: .github/ci.sh generate_buildinfo "${{ github.sha }}" main
```

Comment:
It looks as if `main` is treated as the name of a git branch elsewhere, so perhaps this should be [`${{ github.ref_name }}`](https://docs.github.com/en/actions/reference/workflows-and-actions/contexts#github-context)? Or at least `master`, since this repo uses `master` as the main branch?

(Same comment applies to the `crux-llvm` workflow.)